awx/main/management/commands/run_wsrelay.py

```diff
@@ -101,8 +101,9 @@ def handle(self, *arg, **options):
             migrating = bool(executor.migration_plan(executor.loader.graph.leaf_nodes()))
             connection.close()  # Because of async nature, main loop will use new connection, so close this
         except Exception as exc:
-            logger.warning(f'Error on startup of run_wsrelay (error: {exc}), retry in 10s...')
```

Comment:
Is this an ERROR or WARNING, as the logger function and message don't match?

Do you have an example of `exc`, just wondering what level of detail we will put in the logs.